integration-tests/src/test/groovy/com/okta/sdk/tests/it/ApplicationsIT.groovy

```diff
@@ -516,4 +476,25 @@ class ApplicationsIT extends ITSupport {
         AppUser readAppUser = app.getApplicationUser(appUser2.getId())
         assertThat readAppUser.getCredentials().getUserName(), equalTo("updated-"+user2.getProfile().getEmail())
     }
+
+    @Test
+    void testOAuth2Flow() {
+
+        Client client = Clients.builder()
+            .setOrgUrl("https://java-sdk.oktapreview.com")
+            .setCacheManager(new DisabledCacheManager())
+            .setAuthenticationScheme(AuthenticationScheme.OAUTH2)
+            .setAuthorizationMode("PrivateKey")
+            .setScopes(Arrays.asList("okta.apps.read", "okta.apps.manage"))
+            .setKeyFilePath(getClass().getResource("/privateKey.pem").file)
+            .setAlgorithm("RS256")
+            .setClientId("0oaqqbwsfpD39wH8B0h7")
```

Comment:
I don't think we should hard code the client id. We should extract it from yaml config/env var.

Reply:
Yeah, will update it.

Reply:
This test is probably better suited for a Unit Test, the integration tests when run with a Private Key should be running all this logic already